### PR TITLE
Doc refresh

### PR DESCRIPTION
**Pull request by Rossb0b**

### CODE REVIEW

**Comment by Rossb0b** [comment]
@all-contributors please add @Rossb0b for code

**Comment by Rossb0b** [comment]
@all-contributos please add @Rossb0b for doc

**Comment by Rossb0b** [comment]
@all-contributos please add @Rossb0b for test

File: README.md [hunk 1]
```diff
@@ -1,15 +1,34 @@
...
+
+<p align="center">
+  Node lib to converts HTML with property binding or url to PDF files.
+</p>
+
+<p align="center">
+    <a href="https://github.com/MyUnisoft/events"><img src="https://img.shields.io/github/package-json/v/MyUnisoft/events?style=flat-square" alt="npm version"></a>
+    <a href="https://github.com/MyUnisoft/events"><img src="https://img.shields.io/github/license/MyUnisoft/events?style=flat-square" alt="license"></a>
+    <a href="https://github.com/MyUnisoft/events"><img src="https://img.shields.io/github/languages/code-size/MyUnisoft/events?style=flat-square" alt="size"></a>
+    <a href="./SECURITY.md"><img src="https://img.shields.io/badge/Security-Responsible%20Disclosure-yellow.svg?style=flat-square" alt="Responsible Disclosure Policy" /></a>
```
Comment (fraxken) [inline README.md]: A supprimer

**Review by fraxken — CHANGES_REQUESTED** [review]

**Pull request merged**